app/presenters/tslr_claim_csv_row.rb

```diff
@@ -36,6 +46,13 @@ def submitted_at
     model.submitted_at.strftime("%d/%m/%Y %H:%M")
   end
 
+  def sanitize_field(field)
+    return field unless field.present?
```

Comment:
Assuming `return if field.nil?`is what this is meant to be doing? i.e. guard against it not being a string?

Reply:
I intentionally return what came in. If it's an empty string, there's also nothing to do here, but I didn't want to replace that with `nil`. Do you think that's wrong?

Reply:
That's fair enough, in which case we can switch it from an `unless` to an `if field.blank?`

Reply:
Sure.

Reply:
Done.